portfolio/src/main/webapp/admin.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Coding</title>
+    <link rel="stylesheet" href="index.css">
+    <script src="comment.js"></script>
+  </head>
+  <body>
+    <button onclick="deleteComments();">Delete Comments!</button>
```

Comment:
When switching to the POST/REDIRECT/GET or the analogous DELETE/REDIRECT/GET pattern - don't attach an onclick JS handler to the button, just create a form that submits the POST/DELETE request for the button.

An additional benefit of going with that pattern is that you'll have less JS code managing the comments.